notNeededPackages.json

```diff
@@ -5955,6 +5955,10 @@
             "libraryName": "wix-style-react",
             "asOfVersion": "7.47.1"
         },
+        "wojtekmaj__react-datetimerange-picker": {
+            "libraryName": "@wojtekmaj/react-datetimerange-picker",
+            "asOfVersion": "5.0.0"
```

Comment:
This doesnt seems to be `5.0.1` and not `5.0.0`

Reply:
My thinking was:

* External types were meant to be used with package version 4.x, and is not compatible with version 5.x
* No one will care to update the package for 5.0.0 if 5.0.1 fixed the issue already
* If you installed package version 5.0.0, the correct way to get the types working is to apply bugfix, not to install external types

Therefore, I marked is as not needed as of 5.0.0

WDYT?